app/src/main/cpp/skyline/gpu/trait_manager.cpp

```diff
@@ -79,6 +79,11 @@ namespace skyline::gpu {
             enabledFeatures2.unlink<vk::PhysicalDeviceVertexAttributeDivisorFeaturesEXT>();
         }
 
+        if (supportsLastProvokingVertex)
+            FEAT_SET(vk::PhysicalDeviceProvokingVertexFeaturesEXT, provokingVertexLast, supportsLastProvokingVertex)
+        else
+            enabledFeatures2.unlink<vk::PhysicalDeviceProvokingVertexFeaturesEXT>();
```

Comment:
As `VK_EXT_provoking_vertex` has always had the feature structure, I'd say the `EXT_SET` for this should set a local boolean `hasLastProvokingVertexExt` then this should branch off that variable.

(PS: You can make all local variable naming a bit more consistent with `bool hasCustomBorderColorExt{}, hasShaderAtomicInt64Ext{}, hasShaderFloat16Int8Ext{}, hasShaderDemoteToHelperExt{}, hasProvokingVertexExt{};` while you're at it)